### DISCUSSION TITLE
npm version is outdated

### DISCUSSION THREAD

**Discussion by mvargaslandolfi1993** [MichalLytek/type-graphql#1450] [General]
Why is the npm version different from the github version? Are they still in the development stage? I'm trying to use apollo server and integrate it with type graphql but got this error:

npm ERR! Could not resolve dependency:
npm ERR! peer graphql@"^15.3.0" from type-graphql@1.1.1

And checking the latest version on github if it is updated to version 16.6.0

**Comment by MichalLytek** [comment]
> Why is the npm version different from the github version? Are they still in the development stage?

The `v2.0.0-beta.1` release was published to npm when the GraphQL v16 update PR was in progress, to make sure it works with other packages and in other projects.

When the next beta or RC release will be ready and published, I will create release notes for it.
